content/en/docs/overview/singleton-service/configurations/enable-product-white-list.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Enable Product White List"
+date: 2019-12-11T14:14:49+08:00
+draft: false
+---
+
+Singleton Service add a new feature to support the product white list defination, that means only the products that defined in Singleton Service can work well to get translations and get supported language list. It is no impact for *DateTimes / Numbers / Currencies / Plurals / Measurements / DateFields / RegionList* .
```

Comment:
add -> adds
only the products that defined in Singleton Service can work well to get -> only the product in the white list can get ……